apps/xprof_gui/src/xprof_gui_cowboy1_handler.erl

```diff
@@ -96,6 +96,16 @@ handle_req(<<"data">>, Req, State) ->
         end,
     {ok, ResReq, State};
 
+handle_req(<<"get_callees">>, Req, State) ->
+    MFA = get_mfa(Req),
+    Callees = xprof_core_vm_info:get_called_funs(MFA),
```

Comment:
code in `xprof_gui` app should only call module `xprof_core` which is the documented Erlang API, so please add a proxy function in that module.